inf-clojure.el

```diff
@@ -208,6 +209,21 @@ whichever process buffer you want to use.")
 
 (put 'inf-clojure-mode 'mode-class 'special)
 
+(defcustom inf-clojure-repl-flavor 'clojure
```

Comment:
I really think there should be only a default REPL type and some command that let's easily pick the type of REPL you want to create. Otherwise creating both a lumo repl and a clojure repl would be really frustrating.